bin/alluxio-start.sh

```diff
@@ -242,6 +242,12 @@ start_master() {
   fi
 
   if [[ ${ALLUXIO_MASTER_SECONDARY} == "true" ]]; then
+    local keyword="alluxio.master.AlluxioSecondaryMaster"
+    pids=($(ps -Aww -o pid,command | grep -i "[j]ava" | grep ${keyword} | awk '{print $1}'))
+    if [ ${#pids[@]} -ge 1 ]; then
+        echo "secondary master is already running @ $(hostname -f)."
```

Comment:
User facing messages need to start with capital letter. Changed to something similar to`Secondary master @(hostname -f) is already running`